Homework_8_MyAmazingRPG/MyAmazingRPG/src/Main.java

```diff
@@ -0,0 +1,36 @@
+import java.util.Scanner;   //Import scanner
+
+//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
+// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
+public class Main {
+    public static void main(String[] args) {      //створeння персонажів
+        Scanner select = new Scanner(System.in);
+        boolean selected;
+
+        Character firstPerson = new Character("DemonHunter", 40, 17, 3);
+        Character secondPerson = new Character("DarkHorse", 53, 15, 4);
+
+
+         while (firstPerson.isAlive() & secondPerson.isAlive()){
+            System.out.println("Select who beats 'false' = " + firstPerson.name + " or 'true' = " + secondPerson.name);
+            selected = select.nextBoolean();
+
+            if (selected = true) {
+                firstPerson.takeDamage(secondPerson.Attack(firstPerson));
```

Comment:
if  `.attack()` method included other person taking damage automatically - the whole construction would be much more simple

`firstPerson.attack(secondPerson)`

The inflicted damage would be automatically calculated and `secondPerson.takeDamage()` would be implicitly called inside the `.attack()` method body